kivy/graphics/vertex_instructions_line.pxi

```diff
@@ -1481,20 +1481,38 @@ cdef class SmoothLine(Line):
         return 0
 
     # FIXME: Some artifacts can be observed, depending on the line width,
-    # overdraw_width and radius.
+    # overdraw_width and radius. This occurs due to the way the vertices are
+    # built. It is not noticeable when the alpha value of the color of the
+    # active context is equal to 1. One solution would be to avoid overlapping
+    # vertices.
     cdef void build_smooth(self):
         cdef:
             list p = self.points
+            int close = self._close
```

Comment:
Minor:
Instead of using `close` as a variable name, what about using `can_close`, so it's clearer the intent during the code below?

Reply:
Maybe `must_close_line`, could be a better name?